src/url-replacements.js

```diff
@@ -144,6 +148,82 @@ class UrlReplacements {
     this.set_('SCREEN_HEIGHT', () => {
       return this.win_.screen.height;
     });
+
+    // Returns the time it took to load the whole page. (excludes amp-* elements
+    // that are not rendered by the system yet.)
+    this.set_('PAGE_LOAD_TIME', () => {
```

Comment:
The performance data is available on the page only after the events that they are measuring occur on the page. the load event could happen after a second or more in some cases. If people use a var that depends on that event to occur, the pixel will be sent out only after the event occurs. if that is acceptable, I will add the documentation for these metrics to amp-var-substitutions.md.